### PR TITLE
Change Team Modal UI fix

### PR DESCRIPTION
**Pull request by hakankaradis**
http://recordit.co/Po7Qmos86A
Fixes: #11191

### CODE REVIEW

File: client/app/lib/sidebar/components/resources.coffee [hunk 1]
```diff
@@ -4,7 +4,7 @@ List = require 'app/components/list'
 
 OwnedResourcesList = require './ownedresourceslist'
 SharedResourcesList = require './sharedresourceslist'
-
+SidebarGroupDisabled = require './sidebargroupdisabled'
```
Comment (hakankaradis) [inline client/app/lib/sidebar/components/resources.coffee]: @usirin

File: client/app/lib/changeteam/index.coffee [hunk 2]
```diff
@@ -29,6 +29,6 @@ module.exports = class ChangeTeamView extends kd.ModalView
 
     controller = new ChangeTeamController()
     @addSubView list = controller.getView()
-    controller.on 'ItemsLoaded', ->
+    controller.on 'AllItemsAddedToList', ->
```
Comment (hakankaradis) [inline client/app/lib/changeteam/index.coffee]: kdlistviewcontroller doesn't emit `ItemsLoaded` 
@gokmen according to testfile
https://github.com/koding/koding/blob/master/client/app/test/kodinglistcontroller.test.coffee#L329
It should emit `ItemsLoaded`. Am I missing something or this usage is deprecated?

File: client/app/lib/sidebar/components/resources.coffee (hunk 1 above)
Comment (usirin) [inline client/app/lib/sidebar/components/resources.coffee]: :+1:

**Review by usirin — APPROVED** [review]

File: client/app/lib/changeteam/index.coffee (hunk 2 above)
Comment (gokmen) [inline client/app/lib/changeteam/index.coffee]: @hakankaradis yes, you're right it was emiting that before I removed it in here https://github.com/koding/koding/pull/11150/files#diff-28c8fbaa26775fa401a9d6974d2f1232L122 ✌️ but I didn't understand how the test you provided passed in my PR when I delete that. @szkl, @usirin any ideas?

**Pull request merged**